chameleon.py

```diff
@@ -68,20 +90,28 @@ def stop():
 def _get_cherrypy_page_handler(status_code, content_type, content):
     class Foo(object):
         @cherrypy.expose
+        @cherrypy.tools.json_out()
```

Comment:
It seems like it sets the [content-type as specified](https://bitbucket.org/cherrypy/cherrypy/src/92e4579f029f37ecfff833a71d359a8e914155f3/cherrypy/lib/jsontools.py?at=default#cl-67) in the function arguments. In our case this should take it from the `kwargs` I could add a specific test at some point?